cli/cmd/top.go

```diff
@@ -64,8 +65,8 @@ type tableRow struct {
 const headerHeight = 3
 
 var (
-	columnNames  = []string{"Source", "Destination", "Path", "Count", "Best", "Worst", "Last", "Success Rate"}
-	columnWidths = []int{23, 23, 55, 6, 6, 6, 6, 3}
+	columnNames  = []string{"Source", "Destination", "Method", "Path", "Count", "Best", "Worst", "Last", "Success Rate"}
+	columnWidths = []int{23, 23, 10, 55, 6, 6, 6, 6, 3}
```

Comment:
I'd like for us to keep the top width around 120 columns if possible. I recommend change the "Path" column width from 55 to 37.